config/main.py

```diff
@@ -91,7 +91,7 @@ def _get_breakout_options(ctx, args, incomplete):
 
 def shutdown_interfaces(ctx, del_intf_dict):
     """ shut down all the interfaces before deletion """
-    for intf in del_intf_dict.keys():
+    for intf in list(del_intf_dict.keys()):
```

Comment:
```suggestion
    for intf in del_intf_dict:
```

Reply:
This change was made by the `2to3` tool (see [here](https://www.python.org/dev/peps/pep-0469/#migrating-directly-to-python-3)):

> Note that in many cases d.keys() can be replaced by just d, but the 2to3 migration tool doesn't attempt that replacement.

Plus, keeping `.keys()` makes it clear what the intention is.